pkg/helm/chartproxy/proxy.go

```diff
@@ -78,7 +78,12 @@ func (p *proxy) IndexFile(onlyCompatible bool) (*repo.IndexFile, error) {
 	}
 
 	indexFile := repo.NewIndexFile()
+	var delKeys []string = make([]string, 0, 20)
 	for _, helmRepo := range helmRepos {
+		overwrites := ""
+		if helmRepo.Name == "openshift-helm-charts" {
+			overwrites = "redhat-helm-repo"
+		}
```

Comment:
suggestion: how about that define `helmRepo.OverwriteRepoName()` and put this logic there for better readability?

Reply:
OK

Reply:
Done